lombok.config

```diff
@@ -0,0 +1,2 @@
+# This file is generated by the 'io.freefair.lombok' Gradle plugin
+config.stopBubbling = true
```

Comment:
There are some interesting config options you can add to this file. Make sure to check out the config sections on the lombok doc pages next time you are there.